### DISCUSSION TITLE
include message/attachment size in the analitycs?

### DISCUSSION THREAD

**Discussion by esseeffe** [devmount/third-stats#434] [Ideas]
I'm wondering if it possibile to include also analitycs regarding to message/attachment size.
For example in order to classified sender that waste email space.
Could be affordable?
Thanks

**Comment by devmount** [comment]
Thank you for this suggestion. This is already tracked in #213 which currently is blocked by #376. When I found time to reorganize the UI, all pending new metrics like sizes will be added.
